packages/device_info/lib/device_info.dart

```diff
@@ -119,6 +120,20 @@ class AndroidDeviceInfo {
   /// `false` if the application is running in an emulator, `true` otherwise.
   final bool isPhysicalDevice;
 
+  /// On Android 8.0 (API level 26) and higher versions of the platform,
```

Comment:
We should have a more terse description for this property on its own line for documentation purposes.

Reply:
Any suggestions?

Reply:
Maybe something along the lines of this:

"A unique 64-bit encoded identifier provided by the platform"